darknet_ros/package.xml

```diff
@@ -20,8 +20,14 @@
   <depend>message_generation</depend>
   <depend>darknet_ros_msgs</depend>
   <depend>actionlib</depend>
+  <depend>nodelet</depend>
```

Comment:
Don't you need to add the dependency on image_transport ?

Reply:
Good point!

Reply:
dep on `image_transport` was already in package.xml